<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network Manager D-BUS test</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="refreshBtn">
        <property name="text">
         <string>Przeskanuj</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QCheckBox" name="updateActivation">
        <property name="text">
         <string>Ciągła aktualizacja</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="6">
       <widget class="QQuickWidget" name="quickWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>40</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="resizeMode">
         <enum>QQuickWidget::SizeRootObjectToView</enum>
        </property>
        <property name="source">
         <url>
          <string>file:///home/apcio/Dokumenty/Projekty/D-Bus/NetworkManager/NetworkManager/busyIndicator.qml</string>
         </url>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" rowspan="2" colspan="7">
       <widget class="QTableWidget" name="apList">
        <column>
         <property name="text">
          <string>SSID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>MAC</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Str</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Freq</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Path</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
